### ISSUE TITLE
gamecube retroarch port Info files bug

### ISSUE THREAD

**Issue by Littlemac123** [libretro/RetroArch#13209]
info files are flatout borken on gamecube retroarch

 the problem is retroarch don't understand file types at all and says .nes or .gb is not supported by any cores
so it's to my understanding that it's probably the info files

![IMG_20211108_131533](https://user-images.githubusercontent.com/39867553/140796731-c1a52e68-35e5-4192-8a0f-cc51dc342810.jpg)

![IMG_20211108_131541](https://user-images.githubusercontent.com/39867553/140796736-20441f18-29e2-4a46-8e5d-7245c9eee945.jpg)

as seen here it says there is no cores for my .nes file type i am using quick nes in this screen shot,
and I did set my info cores directory to the directory just in case it wasn't default.

**Comment by Littlemac123** [comment]
this has been broken for a few retroarch updates now, i just assumed i didn't need to report it cause its clearly broken. and after waiting 2 or more retroarch updates I finally reported cause i want this fixed

**Comment by Papy-G** [comment]
You're right, 11-36-2021 nighly doesn't get past the core choice, as no core can recognise its own rom files anymore.

I wonder if someone here really cares about RA GC port.

I'm one of those that would love to help by testing as much as I can, and report back (As I already did with DBorth's standalone emulators, and Theodore (in Libretro)).

**Comment by Papy-G** [comment]
I made two mistakes, you tested quicknes, not nestopia, and I hadn't tried the 1.9.2 yet, now it is done, and it is the last to work, 1.9.3 onwards can't suggest cores for loading.

**Comment by Littlemac123** [comment]
yup newest RetroArch 1.9.14 update still didn't work

**Comment by LibretroAdmin** [comment]
What's the status of this now with 1.11.1?

**Comment by Littlemac123** [comment]
> What's the status of this now with 1.11.1?

it makes me very happy to know your checking on this cause its been broken ever sense you guys got hacked!!!

it seems 1.11.1 was not here https://retroarch.com/?page=platforms for the gamecube build. now I know it should be 1.11.1 but when I checked on the gamecube it says 1.11.0 and still has the same info file error, im using Slot B for my SD card gecko if that matters. im going to now try and get the nightly build and see if that displays as 1.11.1 this time

**Comment by Littlemac123** [comment]
> > What's the status of this now with 1.11.1?
> 
> it makes me very happy to know your checking on this cause its been broken ever sense you guys got hacked!!!
> 
> it seems 1.11.1 was not here https://retroarch.com/?page=platforms for the gamecube build. now I know it should be 1.11.1 but when I checked on the gamecube it says 1.11.0 and still has the same info file error, im using Slot B for my SD card gecko if that matters. im going to now try and get the nightly build and see if that displays as 1.11.1 this time

oh and here is a hash for you. on that current stable release. (1.11.0 - Gambatte (v0.5.0 7e02df6)

**Comment by Littlemac123** [comment]
> What's the status of this now with 1.11.1?

okay the current nightly 2022-10-05_RetroArch build of (1.11.1 - Gambatte (v0.5.0  7e02df6) still errors with "No Cores Available" with loading a GB ROM

**Comment by Papy-G** [comment]
> What's the status of this now with 1.11.1?

Happy to see that the GameCube version eventually gets some attention.

As of 1.12.0 version, still not working. I can choose a game, but then no cores are found for the selected rom.

As a side note, I always wondered why the GameCube version embeds useless GUI items, is it because of the License nature?

**Comment by LibretroAdmin** [comment]
We don't have any current active developers for Gamecube/Wii. Most of the developers that are active on the Nintendo homebrew side are on 3DS/WiiU. 

Come up with these developers and have them look into it and these issues might get fixed. As it is things are like this.
